Content.Server/StationEvents/FakeEvent.cs

```diff
@@ -0,0 +1,43 @@
+﻿using JetBrains.Annotations;
+using Content.Server.GameObjects.EntitySystems.StationEvents;
+using Content.Server.Interfaces.Chat;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.IoC;
+using Robust.Shared.Audio;
+
+namespace Content.Server.StationEvents
+{
+    [UsedImplicitly]
+    public sealed class FakeEvent : StationEvent
+    {
+        public override string Name => "FakeEvent";
+        
+        /// <summary>
+        /// Called once when the station event starts. We only need to announce it's message and we're done!
+        /// </summary>
+        public override void Startup()
+        {
+            Running = true;
+            Occurrences += 1;
+
+            var stationEventSystem = EntitySystem.Get<StationEventSystem>();
+            var randomEvent = stationEventSystem.PickRandomEvent();
+
+            if (randomEvent.StartAnnouncement != null)
+            {
+                EntitySystem.Get<AudioSystem>().PlayGlobal("/Audio/Effects/alert.ogg", AudioParams.Default.WithVolume(-10f));
+                var chatManager = IoCManager.Resolve<IChatManager>();
+                chatManager.DispatchStationAnnouncement(randomEvent.StartAnnouncement);
+            }
+
+            Running = false;
+        }
+        public override void Update(float frameTime)
+        {
+            // no.
+            return;
+        }
+    }
```

Comment:
Yeah handling the announcement is a pain huh.

Alternatively you could assign the StartAnnouncement string randomly, then call the base class, then do whatever else you need (should just be Running = false and the audio).

To complete the effect with authentic audio probably also need to add start and end audios as properties and have the other events check for the property (you'd also need an audio delay property as well) to the base class if you're up to it, otherwise we could just slap a TODO on it and leave it for someone else.

Reply:
Ah, though this is more of a tg based fake event, where the announcement message and sound will play.